client/homebrew/brewRenderer/toolBar/toolBar.jsx

```diff
@@ -185,7 +215,10 @@ const ToolBar = ({ displayOptions, currentPage, totalPages, onDisplayOptionsChan
 					className='previousPage tool'
 					type='button'
 					title='Previous Page(s)'
-					onClick={()=>scrollToPage(pageNum - 1)}
+					onClick={()=>{
+						const rangeOffset = visiblePages.length > 1 ? 1 : 0;
+						scrollToPage(_.min(visiblePages) - visiblePages.length + rangeOffset);
+					}}
 					disabled={pageNum <= 1}
```

Comment:
This line should be updated following your new functionality, so it is disabled when the page range starts from 1, and the next page button when the page range ends in the last page. The CSS will provide the styling when the disabled attribute is set.

Reply:
done and....